docs/rules/terraform_dash_in_data_source_name.md

```diff
@@ -0,0 +1,34 @@
+# terraform_dash_in_data_source_name
```

Comment:
It is good to add links to these rules to the documentation index:
https://github.com/terraform-linters/tflint/blob/b18fd7e6fb8e6381b1307ac5bab16ee08dbec78c/docs/rules/README.md